web/src/components/vis/VisTile.vue

```diff
@@ -14,6 +20,14 @@ v-flex(shrink=1)
   .white.rounded
     v-toolbar.primary.darken-3.top-rounded(dark, flat, dense)
       v-toolbar-title {{ title }}
+      v-spacer
+      v-toolbar-items
+        v-menu(v-if="hasMenu" bottom offset-y left :min-width="150" :close-on-content-click="false")
```

Comment:
My only thought is that it might be preferable to not automatically wrap this in a menu.  If the controls are simple and small, can they be inlined into the toolbar?

IMO fewer clicks is better clicks.  Users aren't usually too curious about kabob menus.

Reply:
The idea would be to put the slot directly under the `v-toolbar-items`, then make each tile specify `v-menu` or whatever else it wants there?

Reply:
Exactly.  The upside is it lays out your controls in a more immediately visible way.   The downside is it has the potential to look less organized or even messy, maybe.   I actually envision these controls looking a bit like toolbar controls a la Microsoft Office

![Screenshot from 2019-09-09 08-00-43](https://user-images.githubusercontent.com/4214172/64529057-0333b500-d2d8-11e9-8af6-7212b484293c.png)

That way the icons are of a unified width and style, but you could use different icons for different plots as appropriate.  I think you'd have a lot of overlap, probably.  Especially if you wanted to do something like allow zoom and pan of the plot.

Now that I think about it, this would be easy to change later.  I'm fine with leaving it as-is now instead of speculating about what additional controls are needed in the future.

Reply:
@jtomeck had a similar thought re not using a dropdown here.

I am going to file a new issue to look at a different/better layout for the menu controls and leave it as a dropdown for now.

Reply:
See 7a47bd71a27770fc3273ae8b09d28d5125a95691 and #172.